lib/src/command/run.dart

```diff
@@ -29,9 +29,8 @@ class RunCommand extends PubCommand {
     argParser.addFlag('enable-asserts', help: 'Enable assert statements.');
     argParser.addFlag('checked', abbr: 'c', hide: true);
     argParser.addMultiOption('enable-experiment',
-        help:
-            'Runs the executable in a VM with the given experiments enabled.\n'
-            '(Enabling experiments will disable source snapshot caching)',
+        help: 'Runs the executable in a VM with the given experiments enabled. '
+            '(Enabling experiments will disable source snapshot caching.)',
```

Comment:
Can we say something simpler like "(Note: pub commands will run slower when using experiments)"?

Reply:
That's very vague. Then I'll rather say nothing.

Reply:
Then say something more specific, but describe the effect (what the user will experience), not the underlying cause of that effect.

Reply:
Maybe -> `(Experiments will skip snapshot cache, this might cause slower startup)`